.github/workflows/run-tests.yml

```
@@ -49,7 +49,7 @@ jobs:
         run: |
           make test_env.run_unit
       - name: Run integration tests
-        if: inputs.run_integration == true
+        if: ${{ always() && inputs.run_integration == true }}
```

Instead of always, should we have `!cancelled()` instead? This will stop the task from running if it's been cancelled. (see below)

---

```suggestion
        if: ${{ !cancelled() && always() && inputs.run_integration == true }}
```
@michelletran-codecov how do you feel about this instead?

---

I think with `!cancelled`, you shouldn't need `always` (see [top comment](https://stackoverflow.com/questions/58858429/how-to-run-a-github-actions-step-even-if-the-previous-step-fails-while-still-f))

---

```suggestions
        if: ${{ !cancelled() && inputs.run_integration == true }}
```